README.md

```diff
@@ -77,39 +82,65 @@ X | ExoComp
 Z | Daily Missions
 ` | Open Alliance Chat - Side of Screen
 
-## Building
+## Contributing / Building
+
+If you wish to contribute to the project, or simply compile the DLL yourself, please see [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## Installation / Runtme
+
+Both the DLL and the toml file must be placed in the STFC game folder which is always located
+at `C:\Games\Star Trek Fleet Command\Star Trek Fleet Command\default\game` as the following
+names:
+
+- community_patch_settings.toml
+- version.dll
+
+The pre-compiled DLL can be downloaded from the official assets hosted at:
```

Comment:
I would point to the Releases section here instead, as that is what will be used in addition to the eventual installer. Just a note to check the releases section is sufficient

Reply:
We should probably follow that up on the #6 PR.